ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController_SwiftUI.swift

```diff
@@ -166,16 +166,17 @@ struct NotificationDemoView: View {
                              actionButtonTitle: actionButtonTitle,
                              actionButtonAction: actionButtonAction,
                              messageButtonAction: messageButtonAction,
-                             overrideTokens: $overrideTokens.wrappedValue ? NotificationOverrideTokens() : nil)
+                             overrideTokens: $overrideTokens.wrappedValue ? notificationOverrideTokens : nil)
     }
 
-    private class NotificationOverrideTokens: NotificationTokens {
-        override var imageColor: DynamicColor {
-            return DynamicColor(light: globalTokens.sharedColors[.orange][.primary])
-        }
-
-        override var horizontalSpacing: CGFloat {
-            return 5.0
-        }
+    private var notificationOverrideTokens: [NotificationTokenSet.Tokens: ControlTokenValue] {
```

Comment:
Nit: Really not important, but if the two token override sets are the same, couldn't we just delete one and make the other not private?

Reply:
I guess so! Unless they were separate for a specific reason.

Reply:
Actually I'm gonna keep them separate because right now the classes are independent, and we haven't answered the "right" way to share code/theming info between our UIKit and SwiftUI demos.